File: lib/layer2_core/dark/e_tertiary_container_dark.dart
import '../../layer1_raw/design_token.dart';
import '../../layer1_raw/dark/system.dart';
import '../../layer1_raw/dark/standard.dart';
import '../../layer1_raw/dark/blue.dart';
import '../../layer1_raw/dark/green.dart';
import '../../layer1_raw/dark/red.dart';
import '../../layer1_raw/dark/nothing.dart';

class ETertiaryContainerDark {
  static DesignToken getToken(dynamic activeTheme) {
    switch (activeTheme.toString().split('.').last) {
      case 'system': return csyd35;
      case 'standard': return cstd35;
      case 'blue': return cbd35;
      case 'green': return cgd35;
      case 'red': return crd35;
      case 'nothing': return cnod35;
      default: return cstd35;
    }
  }
}

File: android/src/main/kotlin/com/example/design_engine/layer2_core/dark/ETertiaryContainerDark.kt
package com.example.design_engine.layer2_core.dark
import com.example.design_engine.R
import com.example.design_engine.layer3_logic.AppTheme
object ETertiaryContainerDark {
    fun getResId(activeTheme: AppTheme): Int {
        return when (activeTheme) {
            AppTheme.SYSTEM -> R.string.csyd35
            AppTheme.STANDARD -> R.color.cstd35
            AppTheme.BLUE -> R.color.cbd35
            AppTheme.GREEN -> R.color.cgd35
            AppTheme.RED -> R.color.crd35
            AppTheme.NOTHING -> R.color.cnod35
        }
    }
}
